linodecli/configuration.py

```diff
@@ -145,11 +148,20 @@ def update(self, namespace, allowed_defaults):
             sys.exit(1)
 
         if self.config.has_section(username) and allowed_defaults:
-            update_dicts = {
-                default_key: self.config.get(username, default_key)
-                for default_key in allowed_defaults
-                if self.config.has_option(username, default_key)
-            }
+            # update_dicts = {
+            #     default_key: self.config.get(username, default_key)
+            #     for default_key in allowed_defaults
+            #     if self.config.has_option(username, default_key)
+            #     }
+            update_dicts = {}
+            for default_key in allowed_defaults:
+                if not self.config.has_option(username, default_key):
+                    continue
+                value = self.config.get(username, default_key)
+                if default_key == 'authorized_users':
```

Comment:
We might want to allow users to specify a list of values for defaults on list fields, either using a string delimiter or using decoded JSON.

Reply:
as mentioned i had some issues reading and writing the list

Reply:
Gotcha, no worries. We can keep this functionality and consider supporting multiple entries down the road :+1: